src/core/boot.js

```diff
@@ -49,7 +49,16 @@ module.exports = (self) => {
 
   if (doInit) {
     self.log('boot:doInit')
-    tasks.push((cb) => self.init(initOptions, cb))
+    tasks.push((cb) => {
+      self._repo.exists((err, exists) => {
```

Comment:
you are still doing the same call twice in the `if (doInit)`. you can just call `repo.exists` and do the if doInit check later down